custom-vu/src/main/kotlin/jces1209/vu/page/CloudIssueNavigator.kt

```diff
@@ -47,6 +50,17 @@ class CloudIssueNavigator(
         .timeout(Duration.ofSeconds(120))
         .build()
 
+    private val filterSubscriptionFalliblePage = FalliblePage.Builder(
+        driver,
+        and(
+            presenceOfElementLocated(By.id("-dialog")),
```

Comment:
could you please exclude to class with @ysemenkov mentioned

Reply:
This isn't JiraTips